.ci/packaging/request_to_obs.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+[ -z "${DEBUG:-}"  ] || set -x
+readonly script_dir=$(dirname $(readlink -f "$0"))
+readonly packaging_dir="${script_dir}/../.."
+
+# Kata branch where packages will be based
+BRANCH=${BRANCH:-master}
+
+# Name of OBS branch to to push
+OBS_BRANCH="${OBS_BRANCH:-testing}"
+
+if [ "${CI:-}" == "true" ] && [ "${GITHUB_PR:-}" != "" ]; then
+	OBS_BRANCH="packaging-PR-${GITHUB_PR}"
+fi
+
+# Push to anywere, variable used by release scripts to push
+PUSH=1
+# dont use release kata image
+BUILD_HEAD=true
+
+if [ "${CI:-}" == "true" ];then
+	SUBPROJECT_TYPE="ci"
+else
+	SUBPROJECT_TYPE="releases"
+fi
+# Name of the OBS subproject under:
+# https://build.opensuse.org/project/subprojects/home:katacontainers
+OBS_SUBPROJECT="${SUBPROJECT_TYPE}:$(uname -m):${OBS_BRANCH}"
+
+export BUILD_HEAD
+export BRANCH
+export OBS_BRANCH
+export OBS_SUBPROJECT
+export PUSH
+
+# azure: Export in all pipeline tasks
+echo "##vso[task.setvariable variable=OBS_SUBPROJECT;]${OBS_SUBPROJECT}"
```

Comment:
Do we need this line? It looks like some sort of magic terminal escape sequence that creates an emoji or something :smile:

Reply:
Indeed,  but the emoji tells azure pieplien to export that variable across its task (stages).  I am afraid I needed to for the next stage that watch the repo until get all the builds results.